modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/repository/capabilities/PortalCapabilityLocatorImpl.java

```diff
@@ -275,4 +276,19 @@ private void _clearLiferayDynamicCapabilities() {
 	@Reference
 	private TrashVersionLocalService _trashVersionLocalService;
 
+	private class PortalCapabilityCacheRegistryItem
+		implements CacheRegistryItem {
+
+		@Override
+		public String getRegistryName() {
+			return _getClassName();
```

Comment:
@GabrielHenriquedev, please hard-coded the className here using the `PortalCapabilityLocatorImpl.class.getName()` statement. Thx